dspace-api/src/test/java/org/dspace/discovery/MockSolrSearchCore.java

```diff
@@ -22,4 +22,8 @@ public void afterPropertiesSet() throws Exception {
         solr = null;
     }
 
+    public void reset() {
+        // This method intentionally left blank.
```

Comment:
I added it because AbstractIntegrationTestWithDatabase calls it.

There used to be two MockSolrSearchCore classes, one in dspace-api and one in dspace-server-webapp.  I think that when I merged them, I kept the wrong one.  I'll try it the other way.

Reply:
Thank you for asking.  This opened an interesting can of worms.  I hadn't copied enough from dspace-server-webapp, which incorrectly allowed this to succeed.